curves/include/curves/helpers.hpp

```diff
@@ -5,132 +5,30 @@
  */
 
 #pragma once
+#include <curves/Curve.hpp>
```

Comment:
We should follow an include standard (see [here](https://github.com/ethz-asl/segmatch/wiki/Contributing-to-SegMatch#header-include-guide)) or at least have a block of includes for the library files of curve